apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -650,6 +653,79 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
                 authenticator.meta.properties.push(scopesMeta);
             }
 
+            // TODO: Need to update below values in the OIDC authenticator metadata API
+            // Set additional meta data if the authenticator is OIDC
+            if (authenticator.id === IdentityProviderManagementConstants.OIDC_AUTHENTICATOR_ID) {
```

Comment:
as a best practice consider having the constant on the right side of the comparison. This will reduce bugs.